types/src/block_executor/config.rs

```diff
@@ -0,0 +1,54 @@
+// Copyright © Aptos Foundation
+
+use crate::on_chain_config::BlockGasLimitType;
+use serde::{Deserialize, Serialize};
+
+#[derive(Clone, Debug)]
+pub struct BlockExecutorLocalConfig {
+    pub concurrency_level: usize,
+}
+
+#[derive(Clone, Debug, Deserialize, Serialize)]
+pub struct BlockExecutorOnchainConfig {
```

Comment:
BlockExecutorOnchainConfigValues to differentiate with the actual OnchainConfigs? Maybe too verbose though

Reply:
hm, good point, I'll see if I can think of a more concise name